### PR TITLE
move test_latest_imagestreams case into openshift tests

### PR DESCRIPTION
**Pull request by pkubatrh**

### CODE REVIEW

**Comment by pkubatrh** [comment]
[test-openshift]

**Comment by ppisar** [comment]
No problem from Perl point of view.

File: test/run-openshift [hunk 1]
```diff
@@ -38,6 +38,9 @@ ct_os_test_template_app ${IMAGE_NAME} \
 # Check the imagestream
 test_perl_imagestream
 
+# check if latest imagestream version is correct
+test_latest_imagestreams
```
Comment (phracek) [inline test/run-openshift:42]: The function `test_latest_imagestreams` is a wrapper around `ct_check_latest_imagestreams`.

File: test/run-openshift [hunk 2]
```diff
@@ -38,6 +38,9 @@ ct_os_test_template_app ${IMAGE_NAME} \
 # Check the imagestream
 test_perl_imagestream
 
+# check if latest imagestream version is correct
```
Comment (phracek) [inline test/run-openshift:41]:
```suggestion
# check if latest imagestream version is correct
test_latest_imagestreams() {
  local result=1
  info "Testing the latest version in imagestreams"
  # Switch to root directory of a container
  pushd "${test_dir}/../.." >/dev/null || return 1
  ct_check_latest_imagestreams
  result=$?
  popd >/dev/null || return 1
  check_result $result
}
```

**Comment by phracek** [comment]
Please remove from `${VERSIONS}/test/run` function `test_latest_imagestreams`.

**Review by phracek — CHANGES_REQUESTED** [review]
Thanks for the PR! `test_latest_imagestreams` function was not moved from ${VERSIONS}/test/run. Please move it.

File: test/run-openshift (hunk 1 above)
Comment (pkubatrh) [reply test/run-openshift:53]: Thanks for spotting this. for some reason I thought this is coming from the test-lib even though it does not have a ct prefix :/ Should be fixed now.

**Comment by pkubatrh** [comment]
[test-openshift]

**Comment by pkubatrh** [comment]
[test-openshift]

**Comment by pkubatrh** [comment]
[test-openshift]

**Review by phracek — APPROVED** [review]
LGTM. Thanks for the PR!. Good work.

**Pull request merged**